test/Algebra/Graph/Test/Label.hs

```diff
@@ -0,0 +1,144 @@
+{-# LANGUAGE OverloadedLists #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.Test.Label
+-- License    : MIT (see the file LICENSE)
+-- Stability  : experimental
+--
+-- Testsuite for "Algebra.Graph.Label".
+-----------------------------------------------------------------------------
+{-
+module Algebra.Graph.Test.Label (
```

Comment:
Why is this commented out? Let's add this header.

Reply:
That's for REPL testing. Will uncomment it in the final draft, before the merge.